### DISCUSSION TITLE
Adobe AIR cannot init Stage3D on Windows 11 Preview version

### DISCUSSION THREAD

**Discussion by nguyenbs** [airsdk/Adobe-Runtime-Support#961] [Q&A]
I've just install Windows 11 on my laptop (16GB RAM, core i7).
I used Harman AIR SDK 33.1.1.406

I've tried to open my game written in Adobe AIR, the app displays first screen well. But when it go to the main screen with Stage3D interactions, the app goes hanging.

Any one else have issue like me with other 3D games?

This is the error trace:

[ERROR] Wed Jul 7 2021 | Rendering:Rendering failed! ArgumentError{name:'ArgumentError',errorID:'3672',message:'Error #3672: Buffer creation failed. Internal error.'} 
ArgumentError: Error #3672: Buffer creation failed. Internal error.
	at flash.display3D::Context3D/createVertexBuffer()
	at alternativa.engine3d.resources::Geometry/upload()[

**Comment by nguyenbs** [comment]
I've update the issue details

**Reply by ajwfrost** [reply]
Hi @nguyenbs  - just checking, do you still have an issue with this? We've checked now with Stage3D on Windows 11 and aren't seeing any problems, but if you're able to reproduce this with the latest Windows 11 release version we can perhaps take a look? Did your laptop previous run it okay with Windows 10, or is this a new machine?
thanks
